src/App.js

```diff
@@ -1,18 +1,16 @@
 import "./App.css";
-import Buttons from "./components/Buttons";
-import Intro from "./components/Intro";
-import Texts from "./components/Texts";
-import Socials from "./components/Socials";
+import { Footer } from "./layout/Footer";
+import { Buttons } from "./components/Buttons";
+import { Header } from "./components/Header";
+import { Texts } from "./components/Texts";
```

Comment:
Great you used named export instead of default ones. Named are always preferred as they are more explicit (some teams may decide to forbid default imports completely).